File: vb/HelperProject/HostComponent.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace HelperProject
	Public Class HostComponent
		Inherits Component

		Private converterComponent1 As ConverterComponent

		Private Sub InitializeComponent()
			Me.converterComponent1 = New HelperProject.ConverterComponent()

		End Sub
	End Class
End Namespace

File: cs/HelperProject/HostComponent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HelperProject {
    public class HostComponent : Component {
        private ConverterComponent converterComponent1;

        private void InitializeComponent() {
            this.converterComponent1 = new HelperProject.ConverterComponent();

        }
    }
}
